files/pulseaudio.service

```diff
@@ -0,0 +1,12 @@
+[Unit]  
```

Comment:
Does the pulseaudio package not ship its own service file?

Reply:
No